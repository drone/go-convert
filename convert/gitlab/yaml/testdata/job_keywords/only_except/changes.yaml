# https://docs.gitlab.com/ee/ci/yaml/#onlychanges--exceptchanges

docker build:
  script: docker build -t my-image:$CI_COMMIT_REF_SLUG .
  only:
    refs:
      - branches
    changes:
      - Dockerfile
      - docker/scripts/*
      - dockerfiles/**/*
      - more_scripts/*.{rb,py,sh}
      - "**/*.json"