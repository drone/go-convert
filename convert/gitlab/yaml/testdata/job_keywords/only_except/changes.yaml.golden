docker build:
    only:
        changes:
            - Dockerfile
            - docker/scripts/*
            - dockerfiles/**/*
            - more_scripts/*.{rb,py,sh}
            - '**/*.json'
        refs:
            - branches
    script:
        - docker build -t my-image:$CI_COMMIT_REF_SLUG .